pathod/language/actions.py

```diff
@@ -52,7 +50,7 @@ def intermediate(self, settings):  # pragma: no cover
 
 
 class PauseAt(_Action):
-    unique_name = None
+    unique_name = None  # type: ignore
```

Comment:
That's not really what the linked issue says. These attributes should have the same type as the base class.